messages/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+
+	//"dont-slack-evil/apphome"
+)
+
+// Response is of type APIGatewayProxyResponse since we're leveraging the
+// AWS Lambda Proxy Request functionality (default behavior)
+//
+// https://serverless.com/framework/docs/providers/aws/events/apigateway/#lambda-proxy-integration
+type Response events.APIGatewayProxyResponse
+
+var slackOauthToken = os.Getenv("SLACK_OAUTH_TOKEN")
```

Comment:
```suggestion
var slackOauthToken = os.Getenv("SLACK_OAUTH_ACCESS_TOKEN")
```
cc https://lesgarshack.slack.com/archives/CUHTQKV9N/p1582956665000700

Reply:
Done